oracle/datadog_checks/oracle/oracle.py

```diff
@@ -88,6 +95,19 @@ def validate_config(self):
         if not self._server or not self._user:
             raise ConfigurationError("Oracle host and user are needed")
 
+        if self._protocol.upper() not in VALID_PROTOCOLS:
+            raise ConfigurationError("Protocol %s is not valid, must either be TCP or TCPS" % self._protocol)
+
+        if self._jdbc_driver and (not self._jdbc_truststore_type or not self._jdbc_truststore_path):
+            raise ConfigurationError(
+                "When connecting via JDBC, both `jdbc_truststore_type` and `jdbc_truststore_path` are required"
```

Comment:
```suggestion
            raise ConfigurationError(
                "Connecting to Oracle via JDBC requires both `jdbc_truststore_type` and `jdbc_truststore_path` configuration options"
```

Also this wasn't the case before, is this supposed to be TCPS specific?

if so maybe rephrase
```

                "TCPS connections to oracle via JDBC requires both `jdbc_truststore_type` and `jdbc_truststore_path` configuration options"
```

Reply:
Good catch, this should only apply for TCPS